doc/source/tune/api_docs/scalability.rst

```diff
@@ -0,0 +1,191 @@
+.. _tune-scalability:
+
+Scalability and overhead benchmarks
+===================================
+
+We conducted a series of microbenchmarks where we tested the scalability of Ray Tune and tried to reason about
+performance overhead we observed. The results from these benchmarks are reflected in the documentation,
+e.g. when we make suggestions on :ref:`how to remove performance bottlenecks <tune-bottlenecks>`.
+
+This page gives an overview over the experiments we did. For each of these experiments, the goal was to
+examine the total runtime of the experiment and address issues when the observed overhead compared to the
+minimal theoretical time was too high (e.g. more than 20% overhead).
+
```

Comment:
Maybe you want to note here that you disabled syncing/extra loggers for these experiments?